src/infrastructure/repositories/offre-emploi-http.repository.ts

```diff
@@ -85,9 +85,10 @@ export class OffresEmploiHttpRepository implements OffresEmploi.Repository {
 
   async getToken(): Promise<string> {
     const TOKEN_EXPIRY_MINUTES = 24
+
     const isTokenExpired =
       this.inMemoryToken.tokenDate &&
-      this.inMemoryToken.tokenDate?.diffNow().as('minute') >
+      this.inMemoryToken.tokenDate.diffNow().as('minute') * -1 >
```

Comment:
Est-ce que `DateService.now().diff(this.inMemoryToken.token).as('minute')` c'est pas plus facile à lire que de faire `* -1` ?